libr/core/cmd_info.c

```diff
@@ -447,7 +447,49 @@ static int cmd_info(void *data, const char *input) {
 			RBININFO ("resources", R_CORE_BIN_ACC_RESOURCES, NULL, 0); 
 			break;
 		case 'r': RBININFO ("relocs", R_CORE_BIN_ACC_RELOCS, NULL, 0); break;
-		case 'd': RBININFO ("dwarf", R_CORE_BIN_ACC_DWARF, NULL, -1); break;
+		case 'd': // "id"
+			if (input[1] == 'p') { // "idp"
+				int mode = false;
+				if (input[2] == '*') { // "idp*"
+					mode = true;
+				}
+				char* filename = strchr (input, ' ');
+				if (filename) {
+					*filename++ = '\0';
+					char* args = strchr (filename, ' ');
+					if (args) {
+						*args = '\0';
+					}
+					int current_fd = r_core_bin_cur (core)->fd;
+					char* r = r_core_cmd_strf (core, "o %s", filename);
```

Comment:
its ok for me to merge this using cmdstr, but should be done with the api if possible in another pr